appveyor.yml

```diff
@@ -77,8 +77,9 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: obvci_install_conda_build_tools.py
     # Workaround for Python 3.4 and x64 bug in latest conda-build.
-    # FIXME: remove this once a new conda-build is released!
-    - cmd: conda install --yes conda-build=1.20.0
+    # FIXME: Remove once there is a release that fixes the upstream issue
+    # ( https://github.com/conda/conda-build/issues/895 ).
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
@ocefpaf - I agree with your concern, but I agree that this is the lesser of the two evils (the other being #447). I'm still holding out for this to be fixed in short order (it *shouldn't* be complex to solve after all).